src/main/java/de/azapps/kafkabackup/restore/offset/OffsetMapper.java

```diff
@@ -0,0 +1,27 @@
+package de.azapps.kafkabackup.restore.offset;
+
+import java.util.Map;
+
+public class OffsetMapper {
+
+  public Long getNewOffset(Map<Long, Long> offsetMap, long maxOriginalOffset, long oldOffset) {
```

Comment:
make it static?

Reply:
can't be static, cause then it would not be possible to mock this service